### DISCUSSION TITLE
Setting up for Trimble catalyst

### DISCUSSION THREAD

**Discussion by chiz-h** [opengisch/QField#3138] [Q&A]
Hi, first post....looking into using a Trimble Catalyst for surveying archaeological trenches. QField has been recommended. Has anyone done this, and what is best way to set up?

Would be wanting to create a project with background mapping and some shapefiles.

Survey would be relatively simple: trench edges with numeric id, spot heights and features with numeric id, all in x/y/z. I guess I need to create a form for the survey attributes

Basic QGIS user, all help appreciated! thanks

**Comment by ThomasDV1984** [comment]
For setup project you find all the necessary documentation on the qfield websites. You can also find some third party use cases which I found really helpful. Qfield can be directly installed on the tablet. Trimble provides an app for the RTK. Make sure mok locations are switched of.

**Reply by chiz-h** [reply]
Thanks, that's really useful. I shal try and find the documentation and work it out! I assume I sort everything in QGIS and then export to QField. I've got Qfield onto my device and will try creating some shapefiles in QGIS that have the right attribute tables

Any idea what the Trimble RTK app is called? What are 'mok locations'?
